aunikd/portfolio
# Fix Resume Link Behavior

I have fixed resume link and now it could be viewed on the new tab when user clicked on it. 

## Description
This pull request addresses an issue with the resume link in the application. The link has been updated to ensure that it now opens in a new tab when clicked, providing a better user experience.

## Changes Made
- Updated the resume link to open in a new tab.

## How to Test
1. Navigate to the home page of the application. 
2. Click on the resume word.
3. Confirm that the resume opens in a new tab.
